src/Core/Services/SendService.cs

```diff
@@ -219,12 +219,12 @@ public async Task<string> SaveWithServerAsync(Send send, EncByteArray encryptedF
                         {
                             response = await LegacyServerSendFileUpload(request, send, encryptedFileData);
                         }
-                        catch (Exception e) 
+                        catch (Exception genE) 
```

Comment:
Would it make more sense to just `catch` with a general `throw`? Since you're not checking the exception, or exception type, or using the exception message, etc. That way the stack-trace is left unadulterated by being re-thrown explicitly.

catch (Exception genE) 
 

 
Would it make more sense to just `catch` with a general `throw`? Since you're not checking the exception, or exception type, or using the exception message, etc. That way the stack-trace is left unadulterated by being re-thrown explicitly.

```c#
                        catch
                        {
                            if (response != default){
                                await _apiService.DeleteSendAsync(response.Id);
                            }
                            throw;
                        }
```

Reply:
yes, sure that's a good improvement; I'll make it happen